lesson-7/cash_machine_http.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class CashMachine
+  FILE_PATH = 'balance.txt'
+  DEFAULT_BALANCE = 100.0
+
+  def initialize
+    @balance = read_balance
+  end
+
+  def deposit(amount)
+    if amount <= 0
+      { error: 'Сумма депозита должна быть больше нуля!' }
+    else
+      @balance += amount
+      save_balance
+      { success: "Депозит успешно выполнен. Новый баланс: #{@balance} руб." }
+    end
+  end
+
+  def withdraw(amount)
+    if amount <= 0
+      { error: 'Сумма вывода должна быть больше нуля!' }
+    elsif amount <= @balance
+      @balance -= amount
+      save_balance
+      { success: "Вывод успешен. Ваш новый баланс: #{@balance} руб." }
+    else
+      { error: 'Денег на счете недостаточно!' }
+    end
+  end
+
+  def balance
+    @balance
+  end
```

Comment:
Лучше использовать аксессер
```suggestion
  attr_reader :balance
```